stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -1729,12 +1730,20 @@ public class ChatClient internal constructor(
             this.callbackExecutor = callbackExecutor
         }
 
-        public override fun buildChatClient(): ChatClient {
+        @InternalStreamChatApi
+        @Deprecated(
```

Comment:
The method name has changed so, I think we don't need this Deprecated annotation here.

Reply:
We're adding this to scare people from picking this method from autocompletion results (deprecation is visible there immediately, while the `@InternalStreamChatApi` annotation only produces an error when already selected and added to the code.